From 5b18c2e80a968411ddd598dc52e4ed5a03c0267b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 05 Dec 2018 14:36:15 +0200
Subject: [PATCH] preboot: move more functions to preboot_ namespace

This makes some order in the functions' names by moving the relevant
functions into 'preboot_' namespace.

Change-Id: If73d2c77ea43d70464563a30de9dbfaf7bdb9692
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot.c b/preboot.c
index e1f442e..d264160 100644
--- a/preboot.c
+++ b/preboot.c
@@ -356,7 +356,7 @@
 #endif
 
 #if defined (DEV_IS_GOYA) && !defined(TARGET_IS_PLDM)
-static int hl_wait_for_bit(unsigned long reg, unsigned long timeout_ns)
+static int preboot_wait_for_bit(unsigned long reg, unsigned long timeout_ns)
 {
 	unsigned int val;
 	unsigned long count = (timeout_ns * SYS_REF_CLOCK) / 1000000000UL;
@@ -370,7 +370,7 @@
 	return 1;
 }
 
-static void hl_wait_for_ext_ld_done(void)
+static void preboot_wait_for_ext_ld_done(void)
 {
 	/*
 	 * Wait for HW FSM to complete 16KB copy to PHY
@@ -385,33 +385,33 @@
 	writel(1, mmPCIE_PHY_PHYX_SRAM_EXT_LD_DONE_3);
 }
 
-static void hl_wait_for_stable_pclk(void)
+static void preboot_wait_for_stable_pclk(void)
 {
 	int ret;
 
 	while (1) {
 		ret = 0;
 		/* Wait for PCLK stable - check MPLLA [no timeout] */
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_0, 100);
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_1, 100);
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_2, 100);
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_3, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_0, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_1, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_2, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLA_STATE_3, 100);
 		if (!ret)
 			break;
 
 		ret = 0;
 		/* Wait for PCLK stable - check MPLLB [no timeout] */
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_0, 100);
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_1, 100);
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_2, 100);
-		ret |= hl_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_3, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_0, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_1, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_2, 100);
+		ret |= preboot_wait_for_bit(mmPCIE_PHY_PHYX_MPLLB_STATE_3, 100);
 		if (!ret)
 			break;
 	}
 }
 #else
-#define hl_wait_for_ext_ld_done()
-#define hl_wait_for_stable_pclk()
+#define preboot_wait_for_ext_ld_done()
+#define preboot_wait_for_stable_pclk()
 #endif
 
 /* Handle PCIe errata: make sure reset_n de-asserted only after perst */
@@ -422,7 +422,7 @@
 		return;
 	}
 
-	hl_wait_for_ext_ld_done();
+	preboot_wait_for_ext_ld_done();
 
 	/* Wait for PRSTn */
 	while (!readl(mmPSOC_GLOBAL_CONF_PRSTN)) ;
@@ -430,7 +430,7 @@
 	/* Enable FW FSM after PRSTn */
 	writel(1, mmPSOC_GLOBAL_CONF_PCI_FW_FSM);
 
-	hl_wait_for_stable_pclk();
+	preboot_wait_for_stable_pclk();
 
 	while (1) {
 		unsigned int prstn, ltssm, smlh, rdlh, hot_reset;
